starboard/build/config/BUILDCONFIG.gn

```diff
@@ -451,6 +451,88 @@ template("shared_library") {
       }
     }
 
+    # Add an additional evergreen_loader target to every gtest target.
+    if (sb_is_evergreen_compatible && sb_evergreen_compatible_package &&
+        target_name != "loader_app") {
+      original_target_name = target_name
+      data_deps += [ "//third_party/icu:icudata" ]
```

Comment:
We're not currently in a target—why are we setting data_deps? If you want something reusable, name it something else, otherwise do this logic in the one target that uses it